File: LoadEstimation/src/Ticket.py
'''
Created on Jul 11, 2016

@author: moosas1
'''

class Ticket():
    id = 0
    t1 = 0
    t2 = 0
    pr = 0
    type = ""
    
    def __init__(self, param):
        parts = param.split() 
        self.id = parts[0] 
        self.t1 = int(parts[2])
        self.t2 = int(parts[3])
        self.pr = int(parts[7])
        self.type = parts[6]
        

File: Expertise/TicketRouting_NEW/src/GetTicketLoad/Ticket.java
package GetTicketLoad;

public class Ticket {
	String id;
	int t1;
	int t2;
	int pr;
	String type;
	
	public Ticket(String input){
		String[] parts = input.split("\t");
		id = parts[0];
		t1 = Integer.parseInt(parts[2]);
		t2 = Integer.parseInt(parts[3]);
		pr = Integer.parseInt(parts[7]);
		type = parts[6];
	}
}
